python/labbox_ephys/api/_workersession.py

```diff
@@ -76,11 +76,15 @@ def handle_message(self, msg):
             self._feed = kp.load_feed(self._feed_uri)
             for key in ['recordings', 'sortings']:
                 self._subfeed_positions[key] = 0
-                subfeed_name = dict(key=key, documentId=self._document_id)
+                subfeed_name = dict(key=key, workspaceName=self._workspace_name)
                 subfeed = self._feed.get_subfeed(subfeed_name)
-                for m in subfeed.get_next_messages(wait_msec=10):
-                    self._send_message({'type': 'action', 'action': m['action']})
-                    self._subfeed_positions[key] = self._subfeed_positions[key] + 1
+                messages = subfeed.get_next_messages(wait_msec=10)
+                for m in messages:
+                    if 'action' in m:
```

Comment:
This check is needed so we don't crash if the feed has some old or incorrect syntax.